pkg/ddc/jindofsx/transform.go

```diff
@@ -365,6 +365,17 @@ func (e *JindoFSxEngine) transformResources(runtime *datav1alpha1.JindoRuntime,
 		}
 	}
 
+	// mem set request
+	if e.getTieredStoreType(runtime) == 0 {
```

Comment:
Could we make the logic easy to understand if `runtime.Spec.Master.Resources.limits.Memory` and `runtime.Spec.Master.Resources.requests.Memory` not set?